src/SDKs/Reservations/Management.Reservations/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Reserved Instance Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Reserved Instance Resources.")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.1.0.0")]
```

Comment:
@juhee0202 please do not update minor version on assemblyversion, unless you have a very specific reason to do that. We maintain semver rules, so you update AssemblyVersion only when you update the major version, else only AssemblyFileVersion is good enought

Reply:
sure. I reverted back to 1.0.0.0